plugins/modules/nxos_config.py

```diff
@@ -415,9 +415,9 @@ def main():
     mutually_exclusive = [("lines", "src", "replace_src"), ("parents", "src")]
```

Comment:
Some day I would like to see `("parents", "src")` struck from this list, but it doesn't have to happen now if it is more complex than just doing that.

Reply:
Will create FRE for this